fero/client.py

```diff
@@ -329,3 +335,23 @@ def get_preauthenticated(self, url, params=None) -> Union[dict, bytes]:
                 verify=self._verify,
             )
         )
+
+    def impersonate(self, username: str):
+        """Impersonate a user.  Only available to admin users.
+
+        :param user_id: The user id to impersonate
```

Comment:
This documentation is not correct. It looks like it takes a `username` string instead of an id.